docs/src/tutorials/Mixed-integer linear programs/facility_location.jl

```diff
@@ -0,0 +1,264 @@
+# Copyright 2017, Iain Dunning, Joey Huchette, Miles Lubin, and contributors    #src
+# This Source Code Form is subject to the terms of the Mozilla Public License   #src
+# v.2.0. If a copy of the MPL was not distributed with this file, You can       #src
+# obtain one at https://mozilla.org/MPL/2.0/.                                   #src
+
+# # Facility Location
+
+# It was originally contributed by Mathieu Tanneau (@mtanneau) and
+# Alexis Montoison (@amontoison).
+
+using Random
+using LinearAlgebra
```

Comment:
Prefer `import Package` for everything except JuMP, so that it is obvious to the reader which functions come from packages outside JuMP.

Reply:
perfect, makes total sense!